gammapy/datasets/spectrum.py

```diff
@@ -314,8 +316,8 @@ def read(cls, filename, format="ogip", checksum=False, name=None, **kwargs):
         ----------
         filename : `~pathlib.Path` or str
             OGIP PHA file to read.
-        format : {"ogip", "ogip-sherpa", "gadf"}
-            Format to use. Default is "ogip".
+        format : {"ogip", "ogip-sherpa", "gadf", None}, optional
```

Comment:
```suggestion
        format : {"ogip", "gadf"}, optional
```
technically `ogip-sherpa` (for energy in keV) is not an option on read